WebServer/frontend/src/app/video-utils/video-utils.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="row">
+    <button (click)="startRecord(time.value)">Record</button>
+    <input #time
+    (keyup.enter)="startRecord(time.value)"
+    (blur)="startRecord(time.value); time.value='' ">
```

Comment:
Er (blur) ikke nå der klikkes ude for dette input felt? Men er det meningen, at den så bare skal i gang med at optage, hvis man gøre det?
```suggestion
>
```

Reply:
Fixed it